server/src/testFixtures/java/io/crate/testing/JoinPairAssert.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.testing;
+
+
+import static io.crate.testing.Asserts.assertThat;
+
+import org.assertj.core.api.AbstractAssert;
+
+import io.crate.analyze.relations.JoinPair;
+import io.crate.metadata.RelationName;
+
+public class JoinPairAssert extends AbstractAssert<JoinPairAssert, JoinPair> {
+
+    protected JoinPairAssert(JoinPair actual) {
+        super(actual, JoinPairAssert.class);
+    }
+
+    public JoinPairAssert isEqualTo(JoinPair expected) {
+        isNotNull();
+        assertThat(actual.joinType()).isEqualTo(expected.joinType());
+        assertThat(actual.left()).isEqualTo(expected.left());
+        assertThat(actual.right()).isEqualTo(expected.right());
+        assertThat(actual.condition()).isEqualTo(expected.condition());
+        return this;
+    }
+
+    public JoinPairAssert hasPairs(RelationName left, RelationName right) {
```

Comment:
`hasPair` would fit better? (looks like we are checking single pair?)

Reply:
👍